.travis/before_install.sh

```diff
@@ -39,21 +39,15 @@ cd /tmp || die
 [ $TRAVIS_OS_NAME != linux ] || {    
     sudo apt-get update || die
 
-    sudo -H pip install --upgrade pip || die
```

Comment:
Since my pip is already the latest, I don't have an easy way to try it. Just wondering, does the next line need sudo?